tests/test_enable_disable.py

```diff
@@ -0,0 +1,46 @@
+from random import choice
+
+from sphinxlint import main
+
+BULLET = "- "
+
+
+def test_default(capsys):
+    main(["sphinxlint", "--list"])
+    out, _err = capsys.readouterr()
+    assert out.count(BULLET) > 10
```

Comment:
```suggestion
    bullets = sum(1 for line in out.splitlines() if line.startswith(BULLET))
    assert bullets > 10
```
This should be more accurate.  You could also factor it out into a `count_bullets` function and used it below.

Reply:
Tried it with `re` instead, you'll tell me what you think.